src/deno_json/mod.rs

```diff
@@ -705,11 +713,20 @@ pub trait DenoJsonCache {
   fn set(&self, path: PathBuf, deno_json: ConfigFileRc);
 }
 
+#[derive(Debug, Error, JsError)]
```

Comment:
Q: when can this error be returned to JS?

Reply:
This is lazily computed now, so maybe in a dynamic import when resolving jsx config.